megatron/model/gpt2_model.py

```diff
@@ -171,17 +165,16 @@ def __init__(self, num_tokentypes=0, parallel_output=True, topology=None):
                                             args.hidden_dropout,
                                             self.init_method,
                                             self.num_tokentypes,
-                                            args.sinusoidal_pos_emb,
                                             tied_weight_attr='word_embeddings_weight'))
         else:
             self.specs.append(LayerSpec(EmbeddingPipe,
-                                self.hidden_size,
-                                args.padded_vocab_size,
-                                args.max_position_embeddings,
-                                args.hidden_dropout,
-                                self.init_method,
-                                self.num_tokentypes,
-                                args.sinusoidal_pos_emb))
+                                        self.hidden_size,
+                                        args.padded_vocab_size,
+                                        args.max_position_embeddings,
+                                        args.hidden_dropout,
+                                        self.init_method,
+                                        self.num_tokentypes))
+
```

Comment:
Is here a `args.pos_emb == "sinusoidal"` missing instead of the previous `args.sinusoidal_pos_emb`?

Reply:
nope, i removed it and instead we use get_args() in the embedding layer  https://github.com/EleutherAI/gpt-neox/pull/147/files/64d0683612f65be7b443e9362a5df34d0d93e92b